Simulator/player.py

```diff
@@ -844,7 +852,9 @@ def return_item_from_bench(self, x):
             if self.bench[x].items:
                 # thiefs_glove_loc_always needs to be cleared even if there's not enough room on bench
                 if self.bench[x].items[0] == 'thiefs_gloves':
-                    self.thiefs_glove_loc.remove([x, -1])
+                    if x in self.thiefs_glove_loc:

```

Comment:
I'll let this be for now but someone else is working on this bug and will fix it on their end.

Reply:
Okay, i'll just revert the changes here then.